action/action-simulator/src/main/java/com/powsybl/action/simulator/tools/ActionSimulatorTool.java

```diff
@@ -232,17 +234,23 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
             checkOptionsInParallel(line);
         }
 
+        ReadOnlyDataSource dataSource = Importers.createDataSource(caseFile);
+        Importer importer = Importers.findImporter(dataSource, context.getShortTimeExecutionComputationManager());
+        if (importer == null) {
+            throw new PowsyblException("No importer found  for case file "  + caseFile);
+        }
+
         //Create observers
         List<LoadFlowActionSimulatorObserver> observers = new ArrayList<>();
         if (!isSubTask) {
-            optionalCaseExporter(line, context.getFileSystem(), DataSourceUtil.getBaseName(caseFile))
+            optionalCaseExporter(line, context.getFileSystem(), importer.getPrettyName(dataSource))
```

Comment:
I think there is a regression here: the case exporter exports IIDM files. If you get a pretty name from the importer, do you export in the same format than the input file?